internal/commands/build.go

```diff
@@ -111,7 +111,11 @@ func Build(logger logging.Logger, cfg config.Config, packClient PackClient) *cob
 				logger.Warn("Using untrusted builder with volume mounts. If there is sensitive data in the volumes, this may present a security vulnerability.")
 			}
 
-			pullPolicy, err := pubcfg.ParsePullPolicy(flags.Policy)
+			stringPolicy := flags.Policy
```

Comment:
I have tried adding this new pattern (check for `--pull-policy` and if it's not set as a parameter, use the configured PullPolicy), but I may have missed some places where it should be changed.